_data/sites.json

```diff
@@ -3371,6 +3371,16 @@
         ]
     },
 
+    {
+        "name": "e621",
+        "url": "https://e621.net/maintenance/user/deletion",
+        "difficulty": "impossible",
+        "notes": "Even if there is a delete option, the site states, that your account will only be changed to a generic name and locked / blanked. Your contributions (submissions, comments, etc) will remain.",
```

Comment:
```suggestion
        "notes": "Even though there is a delete option, the site states it will only change your account to a generic name and lock it. Your contributions (submissions, comments, etc) will remain.",
```